app/controllers/reviews_controller.rb

```diff
@@ -26,11 +26,17 @@ def edit
   def update
     review = Review.find(params[:id])
     review.update(review_params)
-    # require "pry"; binding.pry
     review.save
     redirect_to "/shelters/#{review.shelter_id}"
   end
 
+  def destroy 
+    review = Review.find(params[:id])
+    shelter = review.shelter_id
+    Review.destroy(params[:id])
```

Comment:
So we needed to define the review with params and we also needed to define the shelter associated to the review being deleted before we could destroy it; so that we had access to the shelter_id